server/bingo/tests.py

```diff
@@ -39,3 +39,24 @@ def test_default_total_completions(self):
             points=15,
         )
         self.assertEqual(challenge.total_completions, 0)
+
+
+class RegisterUserTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_register_user(self):
+        response = self.client.post(
+            '/register/', {'username': 'test_user', 'email': 'test@gmail.com',
+                           'first_name': 'larry', 'last_name': 'bird', 'password': 'SuperSecure123'}
+        )
+
+        self.assertEqual(response.status_code, 201)
+
+    def test_register_user_error(self):
```

Comment:
Could you add a couple more tests to verify that there's an error if you give an invalid email or an insecure password?